server/config.go

```diff
@@ -145,22 +145,21 @@ func adjustInt64(v *int64, defValue int64) {
 
 // Parse parses flag definitions from the argument list.
 func (c *Config) Parse(arguments []string) error {
-	flagArgs := make(map[string]*flag.Flag)
-	flag.Visit(func(flag *flag.Flag) {
```

Comment:
This `Visit()` function visits only those flags that have been set.
Before we `Parse()`, all flags are unset, so we always get an empty `flagArgs`.